client/lib/menu-data/menu-data.js

```diff
@@ -49,15 +52,19 @@ function MenuData() {
  */
 Emitter( MenuData.prototype );
 
-/** Generates a home page menu item
+/**
+ * Generates a home page menu item
  * This is used to inject home page item into pages list
+ *
+ * @param {String} pageNameSuffix - page name suffix
+ * @return {Object} object menu builder
  */
-MenuData.prototype.generateHomePageMenuItem = function ( pageNameSuffix ) {
-	return  {
+MenuData.prototype.generateHomePageMenuItem = function( pageNameSuffix ) {
```

Comment:
yes ... I started to implement Classes here but then I thought it was too much?. I think I will implement Classes though :-D.